src/api/app/views/webui/user/_edit_account.html.haml

```diff
@@ -0,0 +1,20 @@
+- if feature_enabled?(:responsive_ux)
+  - content_for :actions do
+    %li.nav-item
+      - if account_edit_link.present?
+        = link_to(account_edit_link, class: 'nav-link') do
+          %i.fas.fa-plus-square.fa-lg.mr-2
+          Edit your account
+      - else
+        = link_to('#', data: { toggle: 'modal', target: '#edit-modal' }, class: 'nav-link') do
+          %i.fas.fa-plus-square.fa-lg.mr-2
+          Edit your account
```

Comment:
I'm want to refactor those action links. I'm currently thinking in doing it with a partial but I don't know if there's a better solution, as a decorator is not really useful here, because the action links are not tied to a specific model.